packages/docusaurus-plugin-content-docs/src/__tests__/lastUpdate.test.ts

```diff
@@ -69,7 +69,12 @@ describe('getFileLastUpdate', () => {
     const consoleMock = jest
       .spyOn(console, 'warn')
       .mockImplementation(() => {});
-    const tempFilePath = path.join(__dirname, '__fixtures__', '.temp');
+    const tempFilePath = path.join(
```

Comment:
You can reference the implementation in #6949 and use the system's temp folder instead

Reply:
Then it errors with:

```console
not a git repository (or any of the parent directories): .git
```

In your case that was ok, since you needed to create a temp git repo dir, but I think here it's not needed.

Reply:
Creating a git repo isn't the most expensive thing we've done in our tests, so I think it's okay, compared to repeatedly writing files within our own repo. We used to do the same thing for the migration test, but that one was later refactored to use FS mocks instead.

In the near future, we would provide a unified tester interface so all git-related tests will be done within the same temp git repo, but for now, I'm okay with initializing another git repo.

Reply:
Yeah, that seems a bit intractable. I'd rather not disable the rule, since you have to import it cross-module anyways. Can you lift this util to the `jest` folder and add a module name mapper like `"@testing-utils/*": "<rootDir>/jest/utils"`? We will eventually publish a separate package for that.

Reply:
I did it, just to show, but I can revert.

> In the near future, we would provide a unified tester interface so all git-related tests will be done within the same temp git repo

Perhaps we should leave as is until then? If you prefer I can even revert my .gitignore addition.

Reply:
> Can you lift this util to the jest folder and add a module name mapper like "@testing-utils/*": "<rootDir>/jest/utils"? We will eventually publish a separate package for that.

Nice. I will take a look.